router/router_test.go

```diff
@@ -0,0 +1,199 @@
+package router
+
+import (
+	"iotdashboard/controller"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestLoginHandler(t *testing.T) {
+	cases := []struct {
+		method, path, email, pass, csrfC, csrfB string
+		status                                  int
+	}{
+		{"POST", "/login", "user@gmail.com", "S3cure3Pa$$", "123", "123", http.StatusOK},
+		{"POST", "/login11", "user@gmail.com", "S3cure3Pa$$", "123", "123", http.StatusOK},
```

Comment:
I don't understand the purpose of this test. You're testing the handler, not the router / muxer, so the different paths should make no difference.

Reply:
That is a really good point but the request still takes in a path, so I wanted to make sure that I wasn't doing something silly with the path in the function, for example with the `redirectTLS` function